### ISSUE TITLE
Exploitation Types - Main Screen - VoiceOver Does Not Read Exploitation Types for Each Good

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#90]
When VoiceOver reads the list of goods for a country, it does not tell the user what type of exploitation it is such as Child Labor, Forced Labor, Forced Child Labor, or some combination of all or two of them.

**Comment by jfc3-dol** [comment]
Now VoiceOver is reading at least the first if there are more than one type of forced labor along with the goods name first.
